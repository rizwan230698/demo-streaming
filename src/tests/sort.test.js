import React from 'react';
import { render } from 'enzyme';
import Sort from '../components/sort';

test('Should render a Sort Component', () => {
  expect(render(<Sort />)).toMatchSnapshot();
});
