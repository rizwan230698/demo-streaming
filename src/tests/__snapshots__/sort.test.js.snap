// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Sort Component 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "ant-select ant-select-single ant-select-show-arrow",
      "style": "width:180px;margin-bottom:10px",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "ant-select-selector",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "ant-select-selection-search",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "aria-activedescendant": "undefined_list_0",
                  "aria-autocomplete": "list",
                  "aria-controls": "undefined_list",
                  "aria-haspopup": "listbox",
                  "aria-owns": "undefined_list",
                  "autocomplete": "off",
                  "class": "ant-select-selection-search-input",
                  "readonly": "",
                  "role": "combobox",
                  "style": "opacity:0",
                  "unselectable": "on",
                  "value": "",
                },
                "children": Array [],
                "name": "input",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "aria-activedescendant": undefined,
                  "aria-autocomplete": undefined,
                  "aria-controls": undefined,
                  "aria-haspopup": undefined,
                  "aria-owns": undefined,
                  "autocomplete": undefined,
                  "class": undefined,
                  "readonly": undefined,
                  "role": undefined,
                  "style": undefined,
                  "unselectable": undefined,
                  "value": undefined,
                },
                "x-attribsPrefix": Object {
                  "aria-activedescendant": undefined,
                  "aria-autocomplete": undefined,
                  "aria-controls": undefined,
                  "aria-haspopup": undefined,
                  "aria-owns": undefined,
                  "autocomplete": undefined,
                  "class": undefined,
                  "readonly": undefined,
                  "role": undefined,
                  "style": undefined,
                  "unselectable": undefined,
                  "value": undefined,
                },
              },
            ],
            "name": "span",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "attribs": Object {
                "class": "ant-select-selection-item",
              },
              "children": Array [
                Object {
                  "data": "Title (A-Z)",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
          Object {
            "attribs": Object {
              "class": "ant-select-selection-item",
            },
            "children": Array [
              Object {
                "data": "Title (A-Z)",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "span",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {
                "class": "ant-select-selection-search",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "aria-activedescendant": "undefined_list_0",
                    "aria-autocomplete": "list",
                    "aria-controls": "undefined_list",
                    "aria-haspopup": "listbox",
                    "aria-owns": "undefined_list",
                    "autocomplete": "off",
                    "class": "ant-select-selection-search-input",
                    "readonly": "",
                    "role": "combobox",
                    "style": "opacity:0",
                    "unselectable": "on",
                    "value": "",
                  },
                  "children": Array [],
                  "name": "input",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "aria-activedescendant": undefined,
                    "aria-autocomplete": undefined,
                    "aria-controls": undefined,
                    "aria-haspopup": undefined,
                    "aria-owns": undefined,
                    "autocomplete": undefined,
                    "class": undefined,
                    "readonly": undefined,
                    "role": undefined,
                    "style": undefined,
                    "unselectable": undefined,
                    "value": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "aria-activedescendant": undefined,
                    "aria-autocomplete": undefined,
                    "aria-controls": undefined,
                    "aria-haspopup": undefined,
                    "aria-owns": undefined,
                    "autocomplete": undefined,
                    "class": undefined,
                    "readonly": undefined,
                    "role": undefined,
                    "style": undefined,
                    "unselectable": undefined,
                    "value": undefined,
                  },
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {
            "aria-hidden": "true",
            "class": "ant-select-arrow",
            "style": "user-select:none;-webkit-user-select:none",
            "unselectable": "on",
          },
          "children": Array [
            Object {
              "attribs": Object {
                "aria-label": "down",
                "class": "anticon anticon-down",
                "role": "img",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "aria-hidden": "true",
                    "class": "",
                    "data-icon": "down",
                    "fill": "currentColor",
                    "focusable": "false",
                    "height": "1em",
                    "viewBox": "64 64 896 896",
                    "width": "1em",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "d": "M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z",
                      },
                      "children": Array [],
                      "name": "path",
                      "namespace": "http://www.w3.org/2000/svg",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "d": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "d": undefined,
                      },
                    },
                  ],
                  "name": "svg",
                  "namespace": "http://www.w3.org/2000/svg",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "aria-hidden": undefined,
                    "class": undefined,
                    "data-icon": undefined,
                    "fill": undefined,
                    "focusable": undefined,
                    "height": undefined,
                    "viewBox": undefined,
                    "width": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "aria-hidden": undefined,
                    "class": undefined,
                    "data-icon": undefined,
                    "fill": undefined,
                    "focusable": undefined,
                    "height": undefined,
                    "viewBox": undefined,
                    "width": undefined,
                  },
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "aria-label": undefined,
                "class": undefined,
                "role": undefined,
              },
              "x-attribsPrefix": Object {
                "aria-label": undefined,
                "class": undefined,
                "role": undefined,
              },
            },
          ],
          "name": "span",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {
            "aria-hidden": undefined,
            "class": undefined,
            "style": undefined,
            "unselectable": undefined,
          },
          "x-attribsPrefix": Object {
            "aria-hidden": undefined,
            "class": undefined,
            "style": undefined,
            "unselectable": undefined,
          },
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
      Object {
        "attribs": Object {
          "aria-hidden": "true",
          "class": "ant-select-arrow",
          "style": "user-select:none;-webkit-user-select:none",
          "unselectable": "on",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "aria-label": "down",
              "class": "anticon anticon-down",
              "role": "img",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "aria-hidden": "true",
                  "class": "",
                  "data-icon": "down",
                  "fill": "currentColor",
                  "focusable": "false",
                  "height": "1em",
                  "viewBox": "64 64 896 896",
                  "width": "1em",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "d": "M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z",
                    },
                    "children": Array [],
                    "name": "path",
                    "namespace": "http://www.w3.org/2000/svg",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "d": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "d": undefined,
                    },
                  },
                ],
                "name": "svg",
                "namespace": "http://www.w3.org/2000/svg",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "aria-hidden": undefined,
                  "class": undefined,
                  "data-icon": undefined,
                  "fill": undefined,
                  "focusable": undefined,
                  "height": undefined,
                  "viewBox": undefined,
                  "width": undefined,
                },
                "x-attribsPrefix": Object {
                  "aria-hidden": undefined,
                  "class": undefined,
                  "data-icon": undefined,
                  "fill": undefined,
                  "focusable": undefined,
                  "height": undefined,
                  "viewBox": undefined,
                  "width": undefined,
                },
              },
            ],
            "name": "span",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "aria-label": undefined,
              "class": undefined,
              "role": undefined,
            },
            "x-attribsPrefix": Object {
              "aria-label": undefined,
              "class": undefined,
              "role": undefined,
            },
          },
        ],
        "name": "span",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "class": "ant-select-selector",
          },
          "children": Array [
            Object {
              "attribs": Object {
                "class": "ant-select-selection-search",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "aria-activedescendant": "undefined_list_0",
                    "aria-autocomplete": "list",
                    "aria-controls": "undefined_list",
                    "aria-haspopup": "listbox",
                    "aria-owns": "undefined_list",
                    "autocomplete": "off",
                    "class": "ant-select-selection-search-input",
                    "readonly": "",
                    "role": "combobox",
                    "style": "opacity:0",
                    "unselectable": "on",
                    "value": "",
                  },
                  "children": Array [],
                  "name": "input",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "aria-activedescendant": undefined,
                    "aria-autocomplete": undefined,
                    "aria-controls": undefined,
                    "aria-haspopup": undefined,
                    "aria-owns": undefined,
                    "autocomplete": undefined,
                    "class": undefined,
                    "readonly": undefined,
                    "role": undefined,
                    "style": undefined,
                    "unselectable": undefined,
                    "value": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "aria-activedescendant": undefined,
                    "aria-autocomplete": undefined,
                    "aria-controls": undefined,
                    "aria-haspopup": undefined,
                    "aria-owns": undefined,
                    "autocomplete": undefined,
                    "class": undefined,
                    "readonly": undefined,
                    "role": undefined,
                    "style": undefined,
                    "unselectable": undefined,
                    "value": undefined,
                  },
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": Object {
                "attribs": Object {
                  "class": "ant-select-selection-item",
                },
                "children": Array [
                  Object {
                    "data": "Title (A-Z)",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "span",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
            Object {
              "attribs": Object {
                "class": "ant-select-selection-item",
              },
              "children": Array [
                Object {
                  "data": "Title (A-Z)",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "span",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {
                  "class": "ant-select-selection-search",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "aria-activedescendant": "undefined_list_0",
                      "aria-autocomplete": "list",
                      "aria-controls": "undefined_list",
                      "aria-haspopup": "listbox",
                      "aria-owns": "undefined_list",
                      "autocomplete": "off",
                      "class": "ant-select-selection-search-input",
                      "readonly": "",
                      "role": "combobox",
                      "style": "opacity:0",
                      "unselectable": "on",
                      "value": "",
                    },
                    "children": Array [],
                    "name": "input",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "aria-activedescendant": undefined,
                      "aria-autocomplete": undefined,
                      "aria-controls": undefined,
                      "aria-haspopup": undefined,
                      "aria-owns": undefined,
                      "autocomplete": undefined,
                      "class": undefined,
                      "readonly": undefined,
                      "role": undefined,
                      "style": undefined,
                      "unselectable": undefined,
                      "value": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "aria-activedescendant": undefined,
                      "aria-autocomplete": undefined,
                      "aria-controls": undefined,
                      "aria-haspopup": undefined,
                      "aria-owns": undefined,
                      "autocomplete": undefined,
                      "class": undefined,
                      "readonly": undefined,
                      "role": undefined,
                      "style": undefined,
                      "unselectable": undefined,
                      "value": undefined,
                    },
                  },
                ],
                "name": "span",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
          ],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "type": "tag",
        "x-attribsNamespace": Object {
          "aria-hidden": undefined,
          "class": undefined,
          "style": undefined,
          "unselectable": undefined,
        },
        "x-attribsPrefix": Object {
          "aria-hidden": undefined,
          "class": undefined,
          "style": undefined,
          "unselectable": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
      "style": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
      "style": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
